src/modules/site-settings/lib/wordpress-versions.ts

```diff
@@ -0,0 +1,30 @@
+import semver from 'semver';
+
+export const addWpVersionToList = (
+	version: string,
+	options: Array< { label: string; value: string } >
+) => {
+	const customVersion = { label: version, value: version };
+	const currentVer = semver.coerce( version );
+
+	// Being extra cautious here, if the version is not valid, we add it to the end of the list.
+	if ( ! currentVer ) {
+		options.push( customVersion );
+		return;
+	}
+
+	// Find the index to insert the new version according to the semver version, newest first.
+	const indexToInsert = options.findIndex( ( compareVersion ) => {
```

Comment:
WDYT about this my comment?
https://github.com/Automattic/studio/pull/1066#discussion_r1995834327